scripts/environment_json.sh

```diff
@@ -20,7 +20,7 @@ PLATFORM=
 function usage () {
   echo "$0 [-r] [-h]"
   echo "  -r: use prod Sentry keys"
-  echo "  -p: platform (android, ios, osx, browser, or electron)"
+  echo "  -p: platform (android, ios, osx, browser, windows, linux, or dev)"
```

Comment:
Could you clarify or add a TODO to clarify what `dev` is?

Reply:
Good idea. I added a TODO here to clarify that this is a hack for the Electron clients, which currently fail if `environment.json` is not present.